### PR TITLE
media-video/rtmpdump: Move to EAPI 6 and git-r3

### PR DESCRIPTION
**Pull request by bjacquin**
And shrink DESCRIPTION to 80 characters

### CODE REVIEW

**Comment by doughdemon** [comment]
Since the ebuild is stable these changes should be done with a revbump.

**Comment by bjacquin** [comment]
Bump made in late PR

File: media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+multilib_src_compile() {
+	if use ssl ; then
+		if use gnutls ;	then
+			crypto="GNUTLS"
+		elif use polarssl ; then
+			crypto="POLARSSL"
+		else
+			crypto="OPENSSL"
+		fi
+	fi
```
Comment (SoapGentoo) [inline media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild]: why if is this code in `multilib_src_compile()` :frowning:?

File: media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+# the library is LGPL-2.1, the command is GPL-2
+LICENSE="GPL-2 LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~amd64-fbsd ~amd64-linux ~arm ~hppa ~mips ~ppc ~ppc64 ~x86 ~x86-fbsd ~x86-linux"
+IUSE="gnutls polarssl ssl libressl"
+
+DEPEND="ssl? (
+		gnutls? ( >=net-libs/gnutls-2.12.23-r6[${MULTILIB_USEDEP},nettle(+)] )
+		polarssl? ( !gnutls? ( >=net-libs/polarssl-1.3.4[${MULTILIB_USEDEP}] ) )
+		!gnutls? ( !polarssl? ( !libressl? ( >=dev-libs/openssl-1.0.1h-r2[${MULTILIB_USEDEP}] ) libressl? ( dev-libs/libressl ) ) )
```
Comment (SoapGentoo) [inline media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild]: both `openssl` and `libressl` need slot operators and need to depend on `SLOT=0`

**Comment by bjacquin** [comment]
I don't mind doing the requested changes, but they were made by the current maintainer of this package. Should I proceed ?

**Comment by monsieurp** [comment]
@bjacquin please.

**Comment by SoapGentoo** [comment]
@bjacquin In general, QA correctness takes precedence over maintainer preferences.

**Comment by bjacquin** [comment]
All changes made in last PR

**Comment by monsieurp** [comment]
This change does not even build and has been reverted. Do provide a build log next time please. Thanks.

**Comment by bjacquin** [comment]
my bad, src_unpack disappeared in my sync between 9999 and release version

**Comment by bjacquin** [comment]
PR updated in https://github.com/gentoo/gentoo/pull/2331

**Pull request closed without merging**